analysis/wwz/wwz4l.py

```diff
@@ -240,12 +240,12 @@ def process(self, events):
         else:
             raise Exception(f"ERROR: Unknown year {year}.")
 
-        # If this is a 2022 sample, get the era info
```

Comment:
This is relatively compact, but sorry it makes my brain really hurt. Two equals lines in one line? Did not even know you could do that. 

To keep things simpler so we don't have to think while reading the code, I suggest we leave this part of the code alone (just leave it how it was before with setting eras). 

Then right above the trigger check, just add this:
```
era_for_trg_check = era
if not (is2022 or is2023):
    era_for_trg_check = None
```